core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/claim/dao/ClaimDBConstants.java

```diff
@@ -27,6 +27,7 @@
     public static final String DELETE_CLAIM_SQL = "DELETE FROM UM_CLAIM WHERE UM_CLAIM_URI=? AND " +
             "UM_DIALECT_ID=(SELECT UM_ID FROM UM_DIALECT WHERE UM_DIALECT_URI=? " +
             "AND UM_TENANT_ID=?) AND UM_TENANT_ID=?";
+    public static final String DELETE_CLAIM_BY_TENANT_ID_SQL = "DELETE FROM UM_CLAIM WHERE UM_TENANT_ID=?";
```

Comment:
These should not be used from Identity Server. Improved claim manager functionality is moved to the carbon-identity-framework repo. Check if you have some necessary changes there as well.

Reply:
Yes. Identity server uses claim metadata service for claim management. Anyway, check whether we are adding any records to these UM tables while adding a tenant. If not, we can skip this.

Reply:
No records with `UM_CLAIMS, UM_DIALECT, UM_CLAIM_BEHAVIOR, UM_PROFILE_CONFIG` tables which are related to claims in tenant addition. I'm removing them.